content/docs/lists-and-keys.md

```diff
@@ -263,7 +263,7 @@ With the example above, the `Post` component can read `props.id`, but not `props
 
 ### Embedding map() in JSX {#embedding-map-in-jsx}
 
-In the examples above we declared a separate `listItems` variable and included it in JSX:
+In the examples below we declared a separate `listItems` variable and included it in JSX:
```

Comment:
I think it's supposed to be "above" because it's referencing previous examples where we declared `listItems` (which you can see if you searched for "listItems").